manifests/role/app_host/hyrax.pp

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2018 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# nebula::role::app_host::prod
+#
+# Standalone application host including apache & mysql
+#
+# @example
+#   include nebula::role::app_host::standalone
+class nebula::role::app_host::standalone {
```

Comment:
Travis is failing this branch because the filename and role do not match. Also, please make sure the copyright date and comments are correct, then squash it all down to one commit.

Reply:
I see there is a weird self-merge in the history... If a pull / rebase doesn't succeed trivially, do it with `merge --squash`:

```
git branch -m this-will-be-squashed
git checkout -b AEIM-1985-general-solr-puppetization master
git merge --squash this-will-be-squashed
git commit
# write a good, small commit message
git push -f
git branch -D this-will-be-squashed
```

Reply:
Also it seems like that in the duplicate entries the newer ones make the changes, while the older ones don't contain any changes but are in the log. 

I also tried the command sequence and it didn't work for me. Will touch base with you tomorrow.

Reply:
Okay the branch issues seem fixed now. Changes made and squashed and pushed. Waiting for final Travis. Then it will be ready to be merged tomorrow during the day.

Reply:
This is fixed.

Reply:
Now it is fixed. *koff*